library/unwind/src/libunwind.rs

```diff
@@ -33,10 +33,10 @@ pub const unwinder_private_data_size: usize = 2;
 #[cfg(all(target_arch = "x86_64", target_os = "windows"))]
 pub const unwinder_private_data_size: usize = 6;
 
-#[cfg(all(target_arch = "arm", not(all(target_vendor = "apple", not(target_os = "watchos")))))]
+#[cfg(all(target_arch = "arm", not(target_vendor = "apple")))]
 pub const unwinder_private_data_size: usize = 20;
 
-#[cfg(all(target_arch = "arm", all(target_vendor = "apple", not(target_os = "watchos"))))]
+#[cfg(all(target_arch = "arm", target_vendor = "apple"))]
 pub const unwinder_private_data_size: usize = 5;
```

Comment:
I have verified that this is correct by compiling the following C code on different Apple platforms:
```c
#include <unwind.h>
int SIZE_OF_PRIVATE_DATA = (sizeof(_Unwind_Exception) - sizeof(uint64_t) - sizeof(void*)) / sizeof(void*);
```